src/autotune.c

```diff
@@ -164,29 +164,19 @@ static int autotune (hashcat_ctx_t *hashcat_ctx, hc_device_param_t *device_param
 
     const u32 kernel_power_max = device_param->hardware_power * kernel_accel_max;
 
-    int CU_rc;
-    int HIP_rc;
-    int CL_rc;
-
     if (device_param->is_cuda == true)
     {
-      CU_rc = run_cuda_kernel_atinit (hashcat_ctx, device_param, device_param->cuda_d_pws_buf, kernel_power_max);
-
-      if (CU_rc == -1) return -1;
+      if (run_cuda_kernel_atinit (hashcat_ctx, device_param, device_param->cuda_d_pws_buf, kernel_power_max) == -1) return -1;
```

Comment:
Nice I think these were left-overs. If you see more of them, please continue to rewrite the same way.